### PR TITLE
[ci] add SHAs to trivy images

### PR DESCRIPTION
**Pull request by makoscafee**
## Checklist

* [x] I have updated the CHANGELOG of any plugins updated
* [x] I have updated the version file of any plugins updated
* [x] I have added tests wherever appropriate
* [x] I have labeled this PR with a label per plugin updated

### CODE REVIEW

File: plugins/ci/pkg/ci/ci.go [hunk 1]
```diff
@@ -220,30 +220,70 @@ func GetRepoTags(path string) ([]string, error) {
...
+			return "", err
+		}
+		var jsonBody interface{}
+		err = json.Unmarshal(bytes, &jsonBody)
+		if err != nil {
+			return "", err
+		}
+		var sha string
+		configMap := jsonBody.(map[string]interface{})["config"].(map[string]interface{})
+		sha = configMap["Image"].(string)
```
Comment (rbren) [inline plugins/ci/pkg/ci/ci.go:283]: is there any chance this field won't exist? If so we should handle the `nil` case.

File: plugins/ci/cmd/insights-ci/main.go [hunk 2]
```diff
@@ -217,7 +221,9 @@ func getTrivyReport(images []trivymodels.Image, configurationObject models.Confi
 		if !matchedImage && len(repoTags) > 0 {
 			imageRepo := repoTags[0]
 			imageRepo = strings.Split(imageRepo, ":")[0]
+			ID := fmt.Sprintf("%s@%s", imageRepo, sha)
```
Comment (rbren) [inline plugins/ci/cmd/insights-ci/main.go:224]: I think this needs to be `@sha256:`

**Comment by rbren** [comment]
🤔 I think we do need my (old) PR for this one. This will only set SHA for images that were explicitly in the fairwinds-insights.yaml, not images that were found in the Helm/YAML

File: plugins/ci/cmd/insights-ci/main.go (hunk 2 above)
Comment (makoscafee) [reply plugins/ci/cmd/insights-ci/main.go:224]: mmh aparently `config.Image` has this format `sha256:xxxxxx`
```
 "ArgsEscaped": true,
        "Image": "sha256:2bf39c9ea98d819cb507e965ebebdc87123b348d5127282c24782d43be52fc00",
        "Volumes": null,
        "WorkingDir": "",
```

Comment (rbren) [reply plugins/ci/cmd/insights-ci/main.go:224]: aha! good to know, thanks

**Review by rbren — APPROVED** [review]

**Pull request merged**